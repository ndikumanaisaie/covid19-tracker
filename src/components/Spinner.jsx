import React from 'react';
import { RotatingLines } from 'react-loader-spinner';

const Spinner = () => (
  <div className="spinner">
    <RotatingLines />
  </div>
);

export default Spinner;
