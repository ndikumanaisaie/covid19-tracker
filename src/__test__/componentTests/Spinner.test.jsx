import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from '../../components/Spinner';

it('renders correctly the Home component', () => {
  const spinner = renderer.create(<Spinner />).toJSON();
  expect(spinner).toMatchSnapshot();
});
