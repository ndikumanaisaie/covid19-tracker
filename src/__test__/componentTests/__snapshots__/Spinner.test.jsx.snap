// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Home component 1`] = `
<div
  className="spinner"
>
  <svg
    aria-busy={true}
    aria-label="rotating-lines-loading"
    className="sc-gswNZR guDipu"
    data-testid="rotating-lines-svg"
    role="status"
    speed="0.75"
    stroke="#4fa94d"
    viewBox="0 0 48 48"
    width="96"
    xmlns="http://www.w3.org/2000/svg"
  >
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(0, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(30, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(60, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(90, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(120, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(150, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(180, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(210, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(240, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(270, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(300, 24, 24)"
      width="5"
    />
    <polyline
      className="sc-dkrFOg kkRkKu"
      points="24,12 24,4"
      transform="rotate(330, 24, 24)"
      width="5"
    />
  </svg>
</div>
`;
